### DISCUSSION TITLE
No longer able to swap themes in Figma

### DISCUSSION THREAD

**Discussion by darcimm** [twilio-labs/paste#3418] [General] [answered]
Hey Paste team! I just spotted an issue with swapping themes this morning - I no longer see the swap theme button in the bottom right when I select the Twilio paste theme. I'm trying to swap to Evergreen for Segment designs.

<img width="471" alt="image" src="https://github.com/twilio-labs/paste/assets/83974127/72c90088-430f-4a31-be30-09982659b317">

I have tried both enabling and disabling the twilio paste theme. It no longer even shows in the 'Current file' list although it used to previously.

Here's the file I'm working in: https://www.figma.com/file/npHoqZDj3hLxMQ8dbY1eaH/%5BETP%5D-Marketing-Analytics?type=design&node-id=3729-443549&mode=design&t=F5hBv0XaDstJIeRG-4

**Comment by serifluous** [comment, marked as answer]
Hey @darcimm ! This might be an unintended consequence of our updating the library right now with Figma's new variables features. Is the work you're doing right now urgent, or would you be able to hold out until the end of this week when we get the updates all in?

Thankfully, once variables are ready, swapping to the evergreen theme should be way simpler than it is now: https://help.figma.com/hc/en-us/articles/14506821864087#variable_modes

**Reply by darcimm** [reply]
There's no rush! I can do a bulk swap next week or whenever it's ready :)

**Reply by serifluous** [reply]
Hey @darcimm ! We just published the changes that'll make designing in evergreen theme a lot easier.

@loreina's going to do a demo on how the new features work in your design standup on Monday.

**Reply by charleneyjh1** [reply]
Hi Paste team @serifluous , following up this question - I'm able to use the updated Paste library with Evergreen theme(Thank you!!), however it doesn't work on the fonts. Even though I selected Evergreen theme as the variable, it still uses the Twilio fonts. Could you provide some guidance on how to use Evergreen fonts?

**Reply by serifluous** [reply]
Hey! Unfortunately Figma doesn't yet support variables for typography, though they've said it's coming by the end of the year: https://help.figma.com/hc/en-us/articles/4406787442711-Figma-beta-features

That's the trade-off we decided to make for now to make theme swapping and token management a bit easier.

~~fwiw Segment prod also already uses Twilio Sans.~~

**Reply by charleneyjh1** [reply]
Could you confirm are we officially switching to Twilio Sans now for Segment going forward? @serifluous

**Reply by serifluous** [reply]
~~We don't have official guidance. I've just noticed that Segment prod uses Twilio Sans.~~

~~I'd recommend checking with the other Segment designers and engineers on what's intended or official or not.~~

**Reply by charleneyjh1** [reply]
Could you share which products you saw that are using Twilio sans in Segment? 
I can follow-up on whether it's intended or not. Up till now I think most Segment designers are using the Evergreen theme fonts in the design.

**Reply by serifluous** [reply]
Argh, sorry! Big mistake on my part. I had a custom stylesheet on for segment.com this whole time 🤦 🤦 🤦 

Until Figma rolls out typography variables, we're hoping that Segment designers can work with their engineers with the understanding that Twilio Sans in Figma isn't meant to make it into production yet (and can't anyway until Segment swaps the evergreen theme to the twilio theme), or designers can use a plugin like this one to swap fonts in their files: https://www.figma.com/community/plugin/855541165287118497/Fonts-Changer
